_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -224,6 +225,9 @@ mfxStatus VAAPIVideoProcessing::Init(_mfxPlatformAccelerationService* pVADisplay
 
         VAEntrypoint* va_entrypoints = NULL;
         VAStatus vaSts;
+        VAConfigAttrib attrib[1];
```

Comment:
I'd still prefer ```vector``` since it is easy extendable

Reply:
done.